climada/entity/exposures/litpop/litpop.py

```diff
@@ -176,8 +180,7 @@ def set_countries(self, countries, res_arcsec=30, exponents=(1,1),
                           f'year: {reference_year}, financial mode: {fin_mode}, ' \
                           f'exp: {exponents}, admin1_calc: {admin1_calc}'
 
-        Exposures.__init__(
-            self,
+        exp = cls(
```

Comment:
Within the constructor, identify the location where the object initialization takes place and make sure to create a new instance instead of overwriting `self`.